docs/containers-storage.conf.5.md

```diff
@@ -106,6 +106,10 @@ containers/storage supports three keys
     Tells containers/storage where an ostree repository exists that might have
     previously pulled content which can be used when attempting to avoid
     pulling content from the container registry
+  * convert_images = "false" | "true"
+    If set to true, containers/storage will convert images to the new format in order
```

Comment:
Can we be explicit on what "the new format" refers to?  Just to avoid misinterpretation.

Reply:
sure, I'll specify what that means